onnx/api_macros.h

```diff
@@ -0,0 +1,46 @@
+#ifndef ONNX_API_H
```

Comment:
I think better to use `ONNX_API_MACROS_H`
in case some day we have onnx/api.h

Reply:
oops that's because I initially named this file as onnx/api.h